src/OrbitBase/File.cpp

```diff
@@ -231,6 +231,16 @@ ErrorMessageOr<void> RemoveFile(const std::filesystem::path& file_path) {
   return outcome::success();
 }
 
+ErrorMessageOr<void> CreateDirectory(const std::filesystem::path& file_path) {
+  std::error_code error;
+  std::filesystem::create_directories(file_path, error);
```

Comment:
Have you considered `ErrorMessageOr<bool>` as the return type? `create_directories` will return false if the directory already exists. This might be useful in some situations.

Reply:
done.